### PR TITLE
Task/jenkins 42375 creation permissions

### PR DESCRIPTION
**Pull request by cliffmeyers**
# Description
- See [JENKINS-42375](https://issues.jenkins-ci.org/browse/JENKINS-42375).

# Submitter checklist
- [x] Link to JIRA ticket in description, if appropriate.
- [x] Change is code complete and matches issue description
- [x] Appropriate unit or acceptance tests or explanation to why this change has no tests
- [ ] Reviewer's manual test instructions provided in PR description. See Reviewer's first task below.
- [x] Ran Acceptance Test Harness against PR changes.

# Reviewer checklist
- [ ] Run the changes and verified the change matches the issue description
- [ ] Reviewed the code
- [ ] Verified that the appropriate tests have been written or valid explanation given

### CODE REVIEW

**Comment by cliffmeyers** [comment]
creation-permissions-1

**Comment by cliffmeyers** [comment]
ATH passed: https://ci.blueocean.io/blue/organizations/jenkins/ATH-Jenkinsfile/detail/master/1748/pipeline/18

File: blueocean-dashboard/src/main/js/creation/CreatePipeline.js [hunk 1]
```diff
@@ -34,6 +35,11 @@ export default class CreatePipeline extends React.Component {
         }
     }
 
+    _isCreationDisabled() {
+        const user = User.current();
+        return Security.isSecurityEnabled() && user && !user.permissions.pipeline.create();
```
Comment (kzantow) [inline blueocean-dashboard/src/main/js/creation/CreatePipeline.js]: is there a case this should include `!user` ?

Comment (cliffmeyers) [inline blueocean-dashboard/src/main/js/creation/CreatePipeline.js]: This logic was being used in two places, so I refactored into a util in d1dd429. Let me know if you think it's an improvement.

Some of our security checks are using sorta slightly different approaches in terms of logic, so I am open to refining this stuff a bit.

**Comment by cliffmeyers** [comment]
This passed ATH last week:  https://ci.blueocean.io/blue/organizations/jenkins/ATH-Jenkinsfile/detail/master/1777/pipeline/17

**Pull request merged**